src/main/java/teammates/ui/template/FeedbackSessionsForm.java

```diff
@@ -36,15 +53,127 @@
     private List<ElementTag> fsEndTimeOptions;
     private List<ElementTag> gracePeriodOptions;
     
+    private boolean isShowNoCoursesMessage;
     private boolean isSubmitButtonDisabled;
+    private boolean isSubmitButtonVisible;
     
-    private AdditionalSettingsFormSegment additionalSettings;
+    private ElementTag editButtonTags;
+    private FeedbackSessionsAdditionalSettingsFormSegment additionalSettings;
     
     public FeedbackSessionsForm() {
     }
     
-    public void setCourseIdForNewSession(String courseIdForNewSession) {
-        this.courseIdForNewSession = courseIdForNewSession;
+    
+    public static FeedbackSessionsForm getFsFormForExistingFs(PageData data, 
+                                                              FeedbackSessionAttributes newFeedbackSession,
+                                                              FeedbackSessionsAdditionalSettingsFormSegment additionalSettings) {
+        FeedbackSessionsForm fsForm = new FeedbackSessionsForm();
+        
+        fsForm.setFsDeleteLink(
+                   new Url(data.getInstructorFeedbackSessionDeleteLink(
+                           newFeedbackSession.courseId,
+                           newFeedbackSession.feedbackSessionName, 
+                           "")));
+        
+        fsForm.setCopyToLink(new Url(data.getFeedbackSessionEditCopyLink()));
```

Comment:
As this is in `FeedbackSessionsForm`, there shouldn't be a need to use the setter methods, you can just set them directly..
Alternatively, you can create an instance here and call some private non-static method that does these.
```
public static FSF getDefaultFSF() {
    FSF f = new FSF();
    f.setToDefault();
}

private void setToDefault() {
    ...
}
```
It does feel odd passing a `PageData` into here either way, though...